test/test_torch.py

```diff
@@ -15301,7 +15354,9 @@ def test_svd_tall_all_col_maj(self, device, dtype):
         self._test_svd_helper((5, 20), False, True, device, dtype)
 
 class TestTorch(TestCase, _TestTorchMixin):
-    pass
+    def assertEqual(self, *args, **kwargs):
+        kwargs.setdefault('exact_dtype', True)
+        super(TestTorch, self).assertEqual(self, *args, **kwargs)
```

Comment:
This version of the function is slightly different than the others. I suppose you could make a class that inherits from TestCase, "StrictTestCase," which just overrides assertEqual as the above classes do in your current PR, and then inherit these test classes from it? That might be more maintainable.

Reply:
This is a great idea. In fact, I don't even need a subclass, I can just do the toggle on a class member